init.lua

```diff
@@ -1333,20 +1338,162 @@ function torchcraft:get_layer(layer_type, also_enemy)
                                        32, 32, color)
             end
         end
+    elseif layer_type == "minerals" then
```

Comment:
Nice! I'd rather have those layers merged though (possibly showing blue for minerals and green for grass).